src/validation.cpp

```diff
@@ -1541,7 +1541,8 @@ bool Chainstate::IsInitialBlockDownload() const
     if (m_chain.Tip()->nChainWork < m_chainman.MinimumChainWork()) {
         return true;
     }
-    if (m_chain.Tip()->Time() < NodeClock::now() - m_chainman.m_options.max_tip_age) {
+    if (m_chain.Tip()->Time() <
+            std::chrono::time_point_cast<std::chrono::seconds>(NodeClock::now()) - m_chainman.m_options.max_tip_age) {
```

Comment:
Why not simply use `Now<NodeSeconds>()`, which does the same in less code?

Reply:
Good call, fixed.